src/scripts/main.js

```diff
@@ -1,3 +1,39 @@
 'use strict';
 
 // write your code here
+const body = document.querySelector('body');
+const logo = document.querySelector('.logo');
+
+const promise1 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => {
+    resolve();
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => reject(new Error()), 3000);
+});
+
+const addMessage = () => {
+  const succesDiv = document.createElement('div');
+
+  succesDiv.className = 'message';
+  succesDiv.innerText = 'Promise was resolved!';
+  body.appendChild(succesDiv);
+};
```

Comment:
Here, you can modify the function to accept an argument and then append it as text